src/nomad_simulations/schema_packages/model_method.py

```diff
@@ -111,6 +111,468 @@ class ModelMethodElectronic(ModelMethod):
     )
 
 
+class FrozenCore(ArchiveSection):
+    """
+    Section defining the frozen-core approximation settings for molecular electronic-structure methods.
+
+    In the frozen-core approximation, selected inner-shell (core) orbitals are excluded from
+    the orbital optimization or post-SCF correlation treatment, retaining them at their
+    reference-determinant (e.g., Hartree-Fock or Kohn-Sham) values. This significantly
+    reduces the number of 'active' orbitals, lowering computational cost for large systems.
```

Comment:
```suggestion
    reduces the number of 'valence' orbitals, lowering computational cost.
```

I feel like 'active' orbitals may be somewhat confusing for the CAS people.
With pseudopotentials and LAPW, people typically refer to them as 'valence' orbitals